example/notebooks/basic_usage.ipynb

```diff
@@ -425,9 +425,9 @@
     "# define list of explainers we want to use\n",
     "# full list of supported explainers is present in `Explainers` enum class.\n",
     "explainer_list = [\n",
-    "    # ExplainerWithParams(explainer_name=CVClassificationExplainers.CV_GRADIENT_SHAP_EXPLAINER),\n",
-    "    # ExplainerWithParams(explainer_name=CVClassificationExplainers.CV_INPUT_X_GRADIENT_EXPLAINER),\n",
-    "    # ExplainerWithParams(explainer_name=CVClassificationExplainers.CV_INTEGRATED_GRADIENTS_EXPLAINER),\n",
+    "    ExplainerWithParams(explainer_name=CVClassificationExplainers.CV_GRADIENT_SHAP_EXPLAINER, baselines=torch.zeros((batch_size, 3, 224, 224)).to(device)),\n",
```

Comment:
Shouldn't it also work without passing baselines explicitly? Like previously.

Reply:
IMHO this is outside the scope of the PR. I created separate issue #119 for setting default values of baselines.